### PR TITLE
Review Page Styling

### PR DESCRIPTION
**Pull request by xtine**
Get review page to match latest designs, per eregs/notice-and-comment#61:

screen shot 2016-04-05 at 11 07 52 pm
screen shot 2016-04-05 at 11 08 03 pm

(some icons and images missing, to await final approvals)

### CODE REVIEW

File: regulations/templates/regulations/comment-review-chrome.html [hunk 1]
```diff
@@ -48,54 +43,99 @@ <h3 class="toc-type">Table of Contents</h3>
...
       </div>
-      <div id="comment-preview">
-        <% if (!previewLoading) { %>
-          <button class="preview-button">Preview your comment</button>
-        <% } else { %>
-          <button class="preview-button" disabled>Building...</button>
-        <% } %>
+
+      <div class="additional-info">
+        <h4>Include your information (Optional)</h4>
```
Comment (cmc333333) [inline regulations/templates/regulations/comment-review-chrome.html]: Something to ponder: this text is specific to EPA. Does it belong in `notice-and-comment`? Should it be a configuration? We can get the field list from regs.gov, but it doesn't quite match this, so there'd need to be some sort of conversion.

File: regulations/templates/regulations/comment-review-chrome.html [hunk 2]
```diff
@@ -48,54 +43,99 @@ <h3 class="toc-type">Table of Contents</h3>
...
+                          <%= file.name %>
+                        </div>
+                        <a href="<%= file.previewUrl %>" class="attachment-preview">Preview</a>
+                      </li>
+                    <% }) %>
+                  </ul>
+                </div>
+              <% } %>
+              <div class="edit-comment">
+                <?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (cmc333333) [inline regulations/templates/regulations/comment-review-chrome.html]: Just to verify, are these xml tags actually needed?

File: regulations/templates/regulations/comment-review-chrome.html [hunk 3]
```diff
@@ -9,13 +9,8 @@
...
 {% endblock %}
 
-{% block wayfinding %}
-<span class="subpart">{{doc_number}}</span>
-<span id="active-title">Active Title</span>
-{% endblock %}
-
 {% block header-secondary %}
-<span class="effective-date"><strong>Published Date:</strong> A Date</span>
+  {% include "regulations/preamble-header.html" %}
```
Comment (cmc333333) [inline regulations/templates/regulations/comment-review-chrome.html]: :+1:

**Comment by cmc333333** [comment]
Seems good to me, but I'm not well versed in style conventions.

File: regulations/templates/regulations/comment-review-chrome.html (hunk 2 above)
Comment (xtine) [inline regulations/templates/regulations/comment-review-chrome.html]: I just looked up that the xml tag isn't required, so i removed it.

File: regulations/templates/regulations/comment-review-chrome.html (hunk 1 above)
Comment (xtine) [inline regulations/templates/regulations/comment-review-chrome.html]: Yeah I was thinking this too... the input fields at the very least are design placeholders until the functionality gets wired in, but I was thinking what to do if the text was too specific for -site.

**Comment by jmcarp** [comment]
A few issues:
* The "read the proposal" link doesn't seem to do anything
* This patch deletes the part of the template that allows comment previews and shows loading text while they're building
* The submit button also doesn't seem to do anything
* Some of the selectors in `comment.less` are deeply nested, like `#comment-review .comments .edit-comment .write-icon`. I'm not sure of best practices on selector nesting, but I'm wondering if this would be easier to understand with a flatter structure.

@xtine: we can restore preview and submit separately, or do it here--what do you prefer?

**Comment by xtine** [comment]
There are a few functional things that aren't working yet -- my intention is to have have a follow up patch to address those. Looking at the design, I don't see a preview button but I assume that functionality now is the "view your comment as pdf" link. And submit button is disabled until the checkbox is clicked which also needs to be done.

Over nesting is a problem though, I'll try to flatten that out.

**Comment by jmcarp** [comment]
All right, we can restore features in separate patches. I added comment preview / download earlier, so it should be quick for me to put that back. Do you want to think about flattening selectors here or separately? I think this is good to merge.

**Pull request merged**